python/langsmith/client.py

```diff
@@ -7207,3 +7254,27 @@ def convert_prompt_to_anthropic_format(
         return anthropic._get_request_payload(messages, stop=stop)
     except Exception as e:
         raise ls_utils.LangSmithError(f"Error converting to Anthropic format: {e}")
+
+
+def _convert_stored_attachments_to_attachments_dict(data, attachments_key):
+    attachments_dict = {}
```

Comment:
add a comment for the method. Would make `attachments_key` a kwarg to make downstream code more readable